be/src/olap/rowset/segment_v2/inverted_index_fs_directory.cpp

```diff
@@ -353,6 +370,7 @@ void DorisFSDirectory::FSIndexOutput::close() {
         }
     } else {
         LOG(WARNING) << "File writer is nullptr, ignore finalize and close.";
+        _CLTHROWA(CL_ERR_IO, "close file writer error, _writer = nullptr");
```

Comment:
why throw error here?

Reply:
If just print the warn log here, the dat file is written successfully, but the idx is not.